homeassistant/components/soundtouch/media_player.py

```diff
@@ -100,19 +101,19 @@ def setup_platform(hass, config, add_entities, discovery_info=None):
             return
 
         remote_config = {"id": "ha.component.soundtouch", "host": host, "port": port}
-        soundtouch_device = SoundTouchDevice(None, remote_config)
-        hass.data[DATA_SOUNDTOUCH].append(soundtouch_device)
-        add_entities([soundtouch_device])
+        bose_soundtouch_device = SoundTouchDevice(None, remote_config)
```

Comment:
We prefer to call entity instances `entity` and not `device`.

Reply:
Sure thing!